tests/test_entrypoint.py

```diff
@@ -147,27 +158,37 @@ def test_as_string():
     assert len(unknown) > 0
 
 
-def test_as_config():
-    with tempfile.TemporaryDirectory() as cwd:
-        cfg_file = os.path.join(cwd, "config.ini")
-        with open(cfg_file, "w") as f:
-            f.write("\n".join([
-                "[Section]",
-                "name = 'myname'",
-                "int = 3",
-                "list = [4, 5, 6]",
-                "unknown = 'other'",
-            ]))
-
-        # test config as kwarg
-        opt1, unknown1 = paramtest_function(
-            entry_cfg=cfg_file, section="Section"
-        )
-
-        # test config as commandline args
-        opt2, unknown2 = paramtest_function(
-            ["--entry_cfg", cfg_file, "--section", "Section"]
-        )
+def test_as_argv():  # almost identical to above
+    with cli_args("--name", "myname", "--int", "3", "--list", "4", "5", "6",
+                  "--other"):
+        opt, unknown = paramtest_function()
+        assert opt.name == "myname"
+        assert opt.int == 3
+        assert len(opt.list) == 3
+        assert opt.list[1] == 5
+        assert len(unknown) > 0
+
+
+def test_as_config(tmp_output_dir):
```

Comment:
since @fsoubelet made me suffer through it .. pytest tmp_path factory or similar would not be an option?

Reply:
`tmp_path` is used in the fixture `tmp_output_dir` and is the standard option used if `DEBUG=False` (or not present)
see https://github.com/pylhc/generic_parser/blob/bug/26/NoneParameter/tests/conftest.py

Reply:
oh, my bad, didnt see that ..

Reply:
alternative way to do it (and use pytest only) would be to change the `basetmp` via config file. 
But as far as I can see from my tmp-folder, pytest keeps the symlinks but not the actual output after test-run?